src/coreComponents/fileIO/vtk/VTKPolyDataWriterInterface.hpp

```diff
@@ -251,6 +251,31 @@ class VTKPolyDataWriterInterface
   void writeElementFields( vtkSmartPointer< vtkCellData > celldata,
                            ElementRegionBase const & er ) const;
 
+  /*!
```

Comment:
Everywhere else in the code doxygen's comment start with `/**`. Can you make all this file more consistent please?

Reply:
Done, I replaced /*! with /**